specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2017-10-01/powerbidedicated.json

```diff
@@ -655,6 +661,97 @@
               "description": "Operation type: create, update, delete, etc.",
               "type": "string",
               "readOnly": true
+            },
+            "description": {
+              "type": "string",
+              "description": "Localized description of the operation."
+            }
+          }
+        },
+        "origin": {
+          "type": "string",
+          "description": "Origin of the operation."
+        },
+        "properties": {
```

Comment:
nit: all properties should be readonly? if so, maybe explicit specify the read-only flag